cypress/e2e/fee-fine/overdue-fine-returned-triggers.cy.js

```diff
@@ -0,0 +1,375 @@
+            testData.ruleProps = CirculationRules.getRuleProps(response.rulesAsText);
+          });
+
+          cy.login(userData.username, userData.password, {
+            path: SettingsMenu.circulationPatronNoticeTemplatesPath,
+            waiter: NewNoticePolicyTemplate.waitLoading,
+          });
+        });
+    });
+  });
+
+  after('Deleting created entities', () => {
+    UserEdit.changeServicePointPreferenceViaApi(userData.userId, [testData.userServicePoint.id]);
+    CirculationRules.deleteRuleViaApi(testData.baseRules);
+    ServicePoints.deleteViaApi(testData.userServicePoint.id);
+    cy.deleteLoanPolicy(loanPolicyBody.id);
+    NoticePolicyApi.deleteApi(testData.ruleProps.n);
+    OverdueFinePolicy.deleteApi(overdueFinePolicyBody.id);
+    Users.deleteViaApi(userData.userId);
+    PatronGroups.deleteViaApi(patronGroup.id);
+    cy.deleteItem(itemData.itemId);
+    cy.deleteHoldingRecordViaApi(itemData.holdingId);
+    InventoryInstance.deleteInstanceViaApi(itemData.instanceId);
+    PaymentMethods.deleteViaApi(testData.paymentMethod.id);
+    UsersOwners.deleteViaApi(userOwnerBody.id);
+    Location.deleteViaApiIncludingInstitutionCampusLibrary(
+      testData.defaultLocation.institutionId,
+      testData.defaultLocation.campusId,
+      testData.defaultLocation.libraryId,
+      testData.defaultLocation.id
+    );
+    NoticePolicyTemplateApi.getViaApi({ query: `name=${noticeTemplates.returnedUponAt.name}` }).then(
+      (templateId) => {
+        NoticePolicyTemplateApi.deleteViaApi(templateId);
+      }
+    );
+    NoticePolicyTemplateApi.getViaApi({ query: `name=${noticeTemplates.returnedAfterOnce.name}` }).then(
+      (templateId) => {
+        NoticePolicyTemplateApi.deleteViaApi(templateId);
+      }
+    );
+    NoticePolicyTemplateApi.getViaApi({ query: `name=${noticeTemplates.returnedAfterRecurring.name}` }).then(
+      (templateId) => {
+        NoticePolicyTemplateApi.deleteViaApi(templateId);
+      }
+    );
+  });
+
+  it(
+    'C347874 Overdue fine, returned triggers (vega)',
+    { tags: [TestTypes.criticalPath, devTeams.vega] },
+    () => {
+      createPatronNoticeTemplate(noticeTemplates.returnedUponAt);
+      createPatronNoticeTemplate(noticeTemplates.returnedAfterOnce);
+      createPatronNoticeTemplate(noticeTemplates.returnedAfterRecurring);
+
+      cy.visit(SettingsMenu.circulationPatronNoticePoliciesPath);
+      NewNoticePolicy.waitLoading();
+      NewNoticePolicy.startAdding();
+      NewNoticePolicy.checkInitialState();
+      NewNoticePolicy.fillGeneralInformation(noticePolicy);
+      NewNoticePolicy.addFeeFineNotice(selectOptions(noticeTemplates.returnedUponAt));
+      NewNoticePolicy.addFeeFineNotice(selectOptions(noticeTemplates.returnedAfterOnce), 1);
+      NewNoticePolicy.addFeeFineNotice(selectOptions(noticeTemplates.returnedAfterRecurring), 2);
+      NewNoticePolicy.save();
+      NewNoticePolicy.waitLoading();
+      NewNoticePolicy.check(noticePolicy);
```

Comment:
let's rename this method check to "checkPolicyName"

Reply:
done